com.revolutionarygamesstudio.ThriveLauncher.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: com.revolutionarygamesstudio.ThriveLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: thrivelauncher
+separate-locales: false
+rename-icon: thrive-launcher
+finish-args:
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: thrive-launcher
```

Comment:
Please configure f-e-d-c for automatic updates.

Reply:
They haven't updated unappimage in 3 years

Reply:
This is not about unappimage.

Reply:
But you selected unappimage?

Reply:
Also this flatpak isn't using extra-data?

Reply:
no. it's pinned to line 26:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/114441/164092729-543b9f83-8828-45cd-a63c-d381991706be.png">

Reply:
> Also this flatpak isn't using extra-data?

it doesn't need to.

Reply:
@hfiguiere Is it not to check that the extra data still exists because it has to be downloaded on install. Compared to when you have it as a source and it is included in the resulting bundle?

Reply:
There are more use-cases than ensuring extra-data still exists. In this instance, unless you're going to implement your own automation (which is clearly not part of your PR), I want to make sure you as the downstream maintainer will be notified about new releases.